### PR TITLE
Fix indentation CI

### PR DESCRIPTION
**Pull request by kronbichler**
When merging #15842, I did not realize that the PR was not rebased on top of the new indentation/clang-format version, so the CI of the PR did not catch the problem with respect to how we indent now. I got an angry email by github that `master` is failing, which is hopefully fixed by this PR.

### CODE REVIEW

**Review by blaisb — APPROVED** [review]

**Comment by blaisb** [comment]
@kronbichler and @peterrum . Not sure what's the policy here. I doubt this requires the full Jenkins tests suite, so I'll just wait for another approval to merge.

**Review by peterrum — APPROVED** [review]

**Comment by kronbichler** [comment]
I would not run it in these cases. I usually try to look whether the Jenkins jobs will be affected by the change at all, leaving documentation changes or tutorial changes out of the Jenkins setup.

**Review by drwells — APPROVED** [review]

**Pull request merged**
